xrfeitoria/sequence/sequence_blender.pyi

```diff
@@ -11,16 +11,6 @@ class SequenceBlender(SequenceBase):
     @classmethod
     def use_camera(cls, camera: CameraBlender): ...
     @classmethod
-    def import_actor(
-        cls,
-        file_path: PathLike,
-        actor_name: Optional[str] = ...,
-        location: Vector = ...,
-        rotation: Vector = ...,
-        scale: Vector = ...,
-        stencil_value: int = ...,
-    ) -> ActorBlender: ...
-    @classmethod
```

Comment:
- This should not be deleted. The method of inherited class should have a unique typing hint.
- A similar hint in sequence_unreal is needed as well

Reply:
fixed in fba8cf2